src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,54 @@
+package duke.command;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.Storage;
+import duke.exception.DukeException;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.ui.MessageUi;
+
+/**
+ * Represents commands which find tasks in the task list. A FindCommand
+ * object corresponds a valid Ekud command, which can then be executed.
+ */
+
```

Comment:
I think it would be better to standardize the amount of space between documentation and your code to improve readability. 👍